README.md

```diff
@@ -35,13 +36,29 @@ export VISIER_REDIRECT_URI=
 export VISIER_TARGET_TENANT_ID=
 export VISIER_USERNAME=
 export VISIER_PASSWORD=
+
+echo -n "Enter the client secret for client with id $VISIER_CLIENT_ID: "
+read -s secret
+export VISIER_CLIENT_SECRET=$secret
 ```
 
 Source this file in and the environment is ready for using the connector with OAuth2.0 authentication:
 ```sh
 $ source .env
 ```
 
+As the connector supports [python-dotenv](https://pypi.org/project/python-dotenv/), some users may prefer to simply define the assignements directly in `.env` and not source it into the OS environment. The following snippet shows how credentials obtained via `dotenv` can be used by the connector:
```

Comment:
Because the connector supports [python-dotenv] ((https://pypi.org/project/python-dotenv/), some users may prefer to define assignments directly in `.env` instead of sourcing it into the OS environment. In the following snippet, the connector uses credentials obtained with `dotenv`.